conformance/experimental_conformance_test.go

```diff
@@ -135,7 +136,11 @@ func testExperimentalConformance(t *testing.T) {
 }
 
 func writeReport(logf func(string, ...any), report confv1a1.ConformanceReport, output string) error {
-	rawReport, err := yaml.Marshal(report)
+	rawReport, err := json.Marshal(report)
+	if err != nil {
+		return err
+	}
+	rawReport, err = yaml.JSONToYAML(rawReport)
```

Comment:
Because there was no proper conversion, the json tags weren't properly decoded by the library `gopkg.in/yaml.v2`. Now that you pointed me out on this, I tried to use `sigs.k8s.io/yaml` and it properly worked. I reverted this change.